src/main/java/camp/nextstep/edu/rebellion/calc/CalculationOperator.java

```diff
@@ -0,0 +1,39 @@
+package camp.nextstep.edu.rebellion.calc;
+
+import java.util.Arrays;
+import java.util.function.BiFunction;
+
+public enum CalculationOperator {
+    ADD("+", (val1, val2) -> (val1 + val2)),
+    SUB("-", (val1, val2) -> (val1 - val2)),
+    MUL("*", (val1, val2) -> (val1 * val2)),
+    DIV("/", (val1, val2) -> {
+        if(0 == val2) {
```

Comment:
조건문 사용시 다음과 같이 if 와 괄호 사이에 공백을 넣어주시는게 컨벤션이오니 참고 부탁드려요 :)
```suggestion
        if (0 == val2) {
```
https://myeonguni.tistory.com/1596

Reply:
완전 감사합니다 이것도 습관이라고 일관되게 쭉 틀렸네요 꼼꼼히 봐주셔서 감사합니다 🙇